cmd/sentry/sentry/sentry_multi_client.go

```diff
@@ -626,6 +660,14 @@ func (cs *MultiClient) HandleInboundMessage(ctx context.Context, inreq *proto_se
 	}
 }
 
+func (cs *MultiClient) HandlePeerEvent(_ context.Context, event *proto_sentry.PeerEvent, _ direct.SentryClient) error {
+	eventID := event.EventId.String()
+	peerID := ConvertH512ToPeerID(event.PeerId)
+	peerIDStr := hex.EncodeToString(peerID[:])
+	log.Debug(fmt.Sprintf("Sentry peer did %s", eventID), "peer", peerIDStr)
```

Comment:
We can't log this inside Sentry? (anyway api for this is useful).
If you need this log line:
- add logPrefix. For example: log.Debug("[P2P] peer event, "event", eventID, "peer", peerIDStr)
- unsure if peerID is useful for reader of logs, but maybe